code/modules/mob/living/carbon/xenomorph/abilities/defender/defender_powers.dm

```diff
@@ -193,63 +201,67 @@
 /datum/action/xeno_action/activable/fortify/action_activate()
 	..()
 	var/mob/living/carbon/Xenomorph/xeno = owner
-	if(xeno.fortify && xeno.selected_ability != src)
+	var/datum/behavior_delegate/defender_base/defender_delegate = xeno.behavior_delegate
+	if(defender_delegate.fortified && xeno.selected_ability != src)
 		button.icon_state = "template_active"
 
 /datum/action/xeno_action/activable/fortify/action_deselect()
 	..()
 	var/mob/living/carbon/Xenomorph/xeno = owner
-	if(xeno.fortify)
+	var/datum/behavior_delegate/defender_base/defender_delegate = xeno.behavior_delegate
+	if(defender_delegate.fortified)
 		button.icon_state = "template_active"
 
-/datum/action/xeno_action/activable/fortify/proc/fortify_switch(var/mob/living/carbon/Xenomorph/X, var/fortify_state)
-	if(X.fortify == fortify_state)
+/datum/action/xeno_action/activable/fortify/proc/fortify_switch(var/mob/living/carbon/Xenomorph/fender, var/fortify_state)
+	var/datum/behavior_delegate/defender_base/defender_delegate = fender.behavior_delegate
+	if(defender_delegate.fortified == fortify_state)
 		return
 
 	if(fortify_state)
-		to_chat(X, SPAN_XENOWARNING("You tuck yourself into a defensive stance."))
-		if(X.steelcrest)
-			X.armor_deflection_buff += 10
-			X.armor_explosive_buff += 60
-			X.ability_speed_modifier += 3
-			X.damage_modifier -= XENO_DAMAGE_MOD_SMALL
+		to_chat(fender, SPAN_XENOWARNING("You tuck yourself into a defensive stance."))
+		if(defender_delegate.steelcrest_strain)
+			fender.armor_deflection_buff += 10
```

Comment:
I think these numbers should be refactored into datum vars - they are hidden in the code and the numbers are duplicated.